### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by sharunkumar**

### CODE REVIEW

**Comment by sharunkumar** [comment]
:shipit:

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [main]
   workflow_run:
-    workflows: ["Notion and commit changes"]
+    workflows: ['Notion and commit changes']
```
Comment (sharunkumar) [inline .github/workflows/build-and-deploy.yml:7]: I think these changes are done by the github actions extension formatter

File: src/pages/projects/projects.ts [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
-  postLink?: string;
-  demoLinkRel?: string;
-  [key: string]: any;
+	name: string;
+	demoLink: string;
+	tags?: string[];
+	description?: string;
+	postLink?: string;
+	demoLinkRel?: string;
+	[key: string]: any;
```
Comment (sharunkumar) [inline src/pages/projects/projects.ts:8]: whole project needs to be prettier'd

**Review by arunsathiya — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "@astrojs/rss": "^4.0.7",
     "@astrojs/sitemap": "^3.1.6",
     "@astrojs/tailwind": "^5.1.0",
+    "@fastly/js-compute": "0.5.8",
```
Comment (arunsathiya) [inline package.json:21]: I think this is problematic in my local dev environment:

```sh
yarn && yarn run dev
yarn install v1.22.22
warning package.json: No license field
warning devolio@0.0.1: No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @fastly/js-compute@0.5.8: The engine "node" is incompatible with this module. Expected version "^16 || ^18". Got "20.10.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Review by arunsathiya — COMMENTED** [review]
Seeing an issue with local dev since including `@fastly/js-compute`. Do you see the same issue, @sharunkumar, with yarn possibly?

File: package.json (hunk 3 above)
Comment (arunsathiya) [reply package.json:21]: [I looked at one of my prior setups from 7 months ago](https://github.com/sharunkumar/arun-portfolio/blob/16e9adeaa5026a82c650d1bebdea9560d3c034e9/package.json), and this dependency wasn't part of it back then but Fastly's Compute processes were part of the Github workflows.

Comment (sharunkumar) [reply package.json:21]: > I think this is problematic in my local dev environment:
> 
> ```shell
> yarn && yarn run dev
> yarn install v1.22.22
> warning package.json: No license field
> warning devolio@0.0.1: No license field
> [1/4] 🔍  Resolving packages...
> [2/4] 🚚  Fetching packages...
> error @fastly/js-compute@0.5.8: The engine "node" is incompatible with this module. Expected version "^16 || ^18". Got "20.10.0"
> error Found incompatible module.
> info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
> ```

your local node version is incompatible. use something like https://github.com/Schniz/fnm to manage node versions

Comment (sharunkumar) [reply package.json:21]: after installing and setting up fnm, run `fnm use 18`

**Pull request merged**